pkgs/terraform-config-inspect/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-config-inspect";
+  version = "20210427";
+  owner = "davido-nw";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit owner rev;
+    repo = pname;
+    sha256 = "1x9ifpk28r9hldpvig92zsvrl5hxzc0gj75q5xr20zb3jb22q11n";
+  };
+
+  vendorSha256 = "14r0q13hi8x7h6kkb26i8v1ialwk86ykgmn53i7874w1kvmzx2hg";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X github.com/${owner}/${pname}/cmd.Version=${rev}"
+  ];
+
+  meta = with lib; {
```

Comment:
Can we update the metadata to reflect `terraform-config-inspect`?

Reply:
I caught myself on that :derp: Should be reflected now if you refresh.